packages/front-end/components/Experiment/EditPhaseModal.tsx

```diff
@@ -110,6 +115,31 @@ export default function EditPhaseModal({
           button instead.
         </div>
       )}
+
+      {advancedOptionsOpen && (
+        //edit seed
+        <Field
+          label="Seed"
+          type="input"
+          {...form.register("seed")}
+          helpText={
+            <>
+              <strong className="text-danger">Warning:</strong> Changing this
+              will change bucketing for you variations.
```

Comment:
maybe more like "Changing this will re-randomize experiment traffic" (check with the Lukes though)

Reply:
yeah i will check with Luke

Reply:
@lukesonnet

Reply:
@bryce-fitzsimons this was the same language on the Docs BTW

Reply:
"Changing this will re-randomize experiment traffic" is good to me.